### DISCUSSION TITLE
`/studio/page` should support Markdown.

### DISCUSSION THREAD

**Discussion by beedell-roke** [JulianPrieber/littlelink-custom#585] [Feature requests]
https://linksta.cc/studio/page should support Markdown.

**Comment by JulianPrieber** [comment]
https://linksta.cc/studio/page is html

**Reply by beedell-roke** [reply]
> https://github.com/orgs/LinkStackOrg/discussions/585#discussioncomment-7094638

@JulianPrieber, however, it does not allow the user to write HTML - it provides a barebones https://en.wikipedia.org/w/index.php?title=WYSIWYG&oldid=1218766325 editor, which doesn't support things like `<pre>`. I wouldn't mind if it allowed to me to write HTML, but Markdown provides that functionality too anyway.
